weekly/2025/week-23-24.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Week 23 and 24 in Packit
+date: 2025-06-16
+authors: lbarczio
+tags:
+  - 2025-June
+  - 2025
+  - June
+---
+
+## Week 23-24 (June 3rd – June 16th)
+
+- Team was attending and presenting at [Flock to Fedora](https://fedoraproject.org/flock/2025/) and
+  [Devconf.cz](https://www.devconf.info/cz/) conferences. The highlight for us was connecting with so many Packit users in person!
+  We want to thank everyone who attended our sessions and shared their feedback. If you are curious, here is the list of sessions we held:
+  - [Even more release automation talk](https://cfp.fedoraproject.org/flock-to-fedora-2025/talk/VZETQA/)
+  - [What about a better dist-git CI?](https://cfp.fedoraproject.org/flock-to-fedora-2025/talk/TNVBXQ/)
+  - [Smarter package maintenance with Packit (workshop)](https://cfp.fedoraproject.org/flock-to-fedora-2025/talk/JBYMTX/)
+  - [Kanban, our way](https://pretalx.devconf.info/devconf-cz-2025/talk/TLPB3M/)
+  - [Breaking Changes Without Breaking Users: Lessons from Packit 1.0](https://pretalx.devconf.info/devconf-cz-2025/talk/UTGBCF/)
+  - [Breaking Barriers: Cross-Distro CI/CD with Packit and OBS](https://pretalx.devconf.info/devconf-cz-2025/talk/7DESJP/)
+- Packit now exposes specfile_path set in configuration as the environment variable `PACKIT_SPECFILE_PATH` for all Packit actions. ([`packit#2589`](https://github.com/packit/packit/pull/2589))
```

Comment:
```suggestion
- Packit now exposes `specfile_path` set in configuration as the environment variable `PACKIT_SPECFILE_PATH` for all Packit actions. ([`packit#2589`](https://github.com/packit/packit/pull/2589))
```

Does it apply only if  `specfile_path` is explicitly configured or also when it is inferred from `downstream_package_name`? If the latter, I think it should be mentioned.

Reply:
changed